server/app/views/questiontypes/ApplicantQuestionRendererImpl.java

```diff
@@ -48,16 +59,22 @@ private String getRequiredClass() {
 
   protected abstract DivTag renderTag(
       ApplicantQuestionRendererParams params,
-      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors);
+      ImmutableMap<Path, ImmutableSet<ValidationErrorMessage>> validationErrors,
+      ImmutableList<String> ariaDescribedByIds,
+      boolean hasErrors);
```

Comment:
I'm not following why `hasErrors` is needed. Isn't the presence of any key in the `validationErrors` map a sufficient signal that any errors exist?

Reply:
Added a lengthier comment in the other thread